transformation/src/jade-schema/tables/sequencing_activity.table.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "sequencing_activity",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_started",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_ended",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "used_sequence_file_ids",
```

Comment:
This feels off to me. Don't sequencing activities use libraries and generate sequence files?